packages/salesforcedx-vscode-apex/src/messages/i18n.ts

```diff
@@ -103,6 +105,7 @@ export const messages = {
   yes: 'Yes',
   cancel: 'Cancel',
   overwrite: 'Overwrite',
+  merge: 'Manually merge with existing ESR',
   file_exists: 'The file already exists. Do you want to overwrite it?',
```

Comment:
maybe change this to account for new feature?
```suggestion
  file_exists: 'The file already exists. How do you want to proceed?',
```